agent/sighandlers/termination_handler.go

```diff
@@ -87,8 +88,9 @@ func FinalSave(saver statemanager.Saver, taskEngine engine.TaskEngine) error {
 	})
 	go func() {
 		seelog.Debug("Saving state before shutting down")
-		stateSaved <- saver.ForceSave()
```

Comment:
just curious, will `saver.ForceSave()` be used anywhere else after this change?

Reply:
We will be removing ForceSave() implementation from agent codebase once boltdb is completely implemented.